Sources/Apollo/RequestCreator.swift

```diff
@@ -27,6 +27,15 @@ public protocol RequestCreator {
 }
 
 extension RequestCreator {
+  /// Creates a `GraphQLMap` out of the passed-in operation
+  ///
+  /// - Parameters:
+  ///   - operation: The operation to use
+  /// - Returns: The created `GraphQLMap`
+  public func requestBody<Operation: GraphQLOperation>(for operation: Operation) -> GraphQLMap {
+    return requestBody(for: operation, sendOperationIdentifiers: false)
```

Comment:
Why is `sendOperationIdentifiers` hard-coded to `false` here?

Reply:
Because of this change I missed to correct: https://github.com/apollographql/apollo-ios/pull/804/commits/6e4cc7f333d554af00b8845244da46b92cee5c82

Reply:
I would rather force callers to provide that `sendOperationIdentifiers` parameter if they're using the default implementation than create another method with such a similar signature.

Reply:
Will remove the two methods that "removes" the default parameter.